source/dynamicaudio.js

```diff
@@ -29,71 +30,117 @@ DynamicAudio.prototype = {
             self.swf = opts['swf'];
         }
 
-        // Attempt to create an audio element
-        if (typeof Audio !== 'undefined') {
-            self.audioElement = new Audio();
-            if (self.audioElement.mozSetup) {
-                self.audioElement.mozSetup(2, 44100);
-                return;
-            }
-        }
-
-        // Fall back to creating flash player
-        self.audioElement = null;
-        self.flashWrapper = document.createElement('div');
-        self.flashWrapper.id = 'dynamicaudio-flashwrapper-'+self.id;
-        // Credit to SoundManager2 for this:
-        var s = self.flashWrapper.style;
-        s['position'] = 'fixed';
-        s['width'] = s['height'] = '8px'; // must be at least 6px for flash to run fast
-        s['bottom'] = s['left'] = '0px';
-        s['overflow'] = 'hidden';
-        self.flashElement = document.createElement('div');
-        self.flashElement.id = 'dynamicaudio-flashelement-'+self.id;
-        self.flashWrapper.appendChild(self.flashElement);
+        // Attempt to create HTML5 Web audio context
+        try {
+            self.audioContext = new AudioContext();
+        } catch(e) {
```

Comment:
I wonder if there are any valid errors that need handling and displaying here? At the very least, perhaps it could print a debug statement to the console with the error to make this easier to debug in the case that it's a real error. ("Couldn't create AudioContext because X, falling back to flash player")